extension/diffheader.js

```diff
@@ -109,10 +107,9 @@ window.diffFileHeader = (() => {
 	const setup = () => {
 		$(window).on('scroll.diffheader', () => diffHeaderFilename());
 		const onResize = utils.debounce(() => diffHeaderFilename(true), 200);
-		$(window).on('resize', onResize);
+		$(window).on('resize.diffheader', onResize);
```

Comment:
How does `resize.diffheader` work? Can you link to jQuery docs on it? I've never seen this before.